### PR TITLE
Branch About Us

### PR DESCRIPTION
**Pull request by zylee348**

### CODE REVIEW

**Comment by zylee348** [comment]
will still require updating after the PPP has been completed

File: docs/AboutUs.md [hunk 1]
```diff
@@ -9,51 +9,51 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
...
 
 <img src="images/johndoe.png" width="200px">
 
 [[homepage](http://www.comp.nus.edu.sg/~damithch)]
-[[github](https://github.com/johndoe)]
+[[github](https://github.com/Tan-Jin-Waye)]
 [[portfolio](team/johndoe.md)]
 
-* Role: Project Advisor
+* Role: TBC
```
Comment (Tan-Jin-Waye) [inline docs/AboutUs.md:20]: Since this role is TBC, suggest adding back Responsibilities.

**Review by Tan-Jin-Waye — COMMENTED** [review]
Some formatting suggestions.

File: docs/AboutUs.md (hunk 1 above)
Comment (zylee348) [reply docs/AboutUs.md:20]: good point will create new pr

**Pull request merged**